HippUnfoldHCP/PostHippUnfoldHCP.sh

```diff
@@ -0,0 +1,300 @@
+${CARET7DIR}/wb_command -add-to-spec-file ${PhysicalHippUnfoldFolder}/${Subject}.${Mesh}.wb_spec INVALID ${PhysicalHippUnfoldFolder}/${Subject}.HippocampalSubfields.nii.gz
+${CARET7DIR}/wb_command -volume-resample ${PhysicalHippUnfoldFolder}/${Subject}.HippocampalSubfields.nii.gz ${AtlasFolder}/T1w_restore.nii.gz ENCLOSING_VOXEL ${AtlasHippUnfoldFolder}/${Subject}.HippocampalSubfields.nii.gz -warp ${AtlasFolder}/xfms/acpc_dc2standard.nii.gz -fnirt ${AtlasFolder}/T1w_restore.nii.gz
+${CARET7DIR}/wb_command -add-to-spec-file ${AtlasHippUnfoldFolder}/${Subject}.${Mesh}.wb_spec INVALID ${AtlasHippUnfoldFolder}/${Subject}.HippocampalSubfields.nii.gz
+
+#NIFTI Input Volumes
+${CARET7DIR}/wb_command -add-to-spec-file ${PhysicalHippUnfoldFolder}/${Subject}.${Mesh}.wb_spec INVALID ${T1wFolder}/T1w_acpc_dc_restore.nii.gz
+${CARET7DIR}/wb_command -add-to-spec-file ${PhysicalHippUnfoldFolder}/${Subject}.${Mesh}.wb_spec INVALID ${T1wFolder}/T2w_acpc_dc_restore.nii.gz
+${CARET7DIR}/wb_command -add-to-spec-file ${AtlasHippUnfoldFolder}/${Subject}.${Mesh}.wb_spec INVALID ${AtlasFolder}/T1w_restore.nii.gz
+${CARET7DIR}/wb_command -add-to-spec-file ${AtlasHippUnfoldFolder}/${Subject}.${Mesh}.wb_spec INVALID ${AtlasFolder}/T2w_restore.nii.gz
+
+#TODO: Merge Native Meshes, anything with 0pt5mm meshes?
+if [ $Mesh = "2mm" ] ; then
+  ${CARET7DIR}/wb_command -spec-file-merge ${PhysicalHippUnfoldFolder}/${Subject}.${Mesh}.wb_spec ${T1wFolder}/fsaverage_LR32k/${Subject}.MSMAll.32k_fs_LR.wb.spec ${PhysicalHippUnfoldFolder}/${Subject}.${Mesh}.MSMAll.32k.wb_spec #TODO: Don't Hardcode Cortex
+  ${CARET7DIR}/wb_command -spec-file-merge ${AtlasHippUnfoldFolder}/${Subject}.${Mesh}.wb_spec ${AtlasFolder}/fsaverage_LR32k/${Subject}.MSMAll.32k_fs_LR.wb.spec ${AtlasHippUnfoldFolder}/${Subject}.${Mesh}.MSMAll.32k.wb_spec #TODO: Don't Hardcode Cortex
+elif [ $Mesh = "1mm" ] ; then
+  ${CARET7DIR}/wb_command -spec-file-merge ${PhysicalHippUnfoldFolder}/${Subject}.${Mesh}.wb_spec ${AtlasFolder}/${Subject}.MSMAll.164k_fs_LR.wb.spec ${PhysicalHippUnfoldFolder}/${Subject}.${Mesh}.MSMAll.164k.wb_spec #TODO: Don't Hardcode Cortex
+fi
+
+if [ ${Flag} = "On" ] ; then
+  Modality="T1wT2w"
+fi
+
+}
+
+for Modality in T1w T2w T1wT2w ; do 
+  for Mesh in 0p5mm 1mm 2mm ; do #TODO: Native meshes not yet available from HippUnfold: "Native" Folder Name "native" Mesh Name 
+    
+    PhysicalHippUnfoldFolderOut="$PhysicalHippUnfoldDIR/${Modality}_hippunfold"
+    AtlasHippUnfoldolderOut="$AtlasHippUnfoldDIR/${Modality}_hippunfold"
+
+    PhysicalHippUnfoldFolder="$PhysicalHippUnfoldFolderOut/$Mesh"
+    AtlasHippUnfoldFolder="$AtlasHippUnfoldolderOut/$Mesh"
+    
+    mkdir -p ${PhysicalHippUnfoldFolder} ${AtlasHippUnfoldFolder}
+
+    log_Msg "Processing $Modality $Mesh"
+    PostHippUnfold $PhysicalHippUnfoldFolderOut $PhysicalHippUnfoldFolder $AtlasHippUnfoldFolder $T1wFolder $AtlasFolder $Subject $Modality $Mesh
```

Comment:
```suggestion
for LoopModality in T1w T2w T1wT2w ; do 
  for Mesh in 0p5mm 1mm 2mm ; do #TODO: Native meshes not yet available from HippUnfold: "Native" Folder Name "native" Mesh Name 
    
    PhysicalHippUnfoldFolderOut="$PhysicalHippUnfoldDIR/${LoopModality}_hippunfold"
    AtlasHippUnfoldolderOut="$AtlasHippUnfoldDIR/${LoopModality}_hippunfold"

    PhysicalHippUnfoldFolder="$PhysicalHippUnfoldFolderOut/$Mesh"
    AtlasHippUnfoldFolder="$AtlasHippUnfoldolderOut/$Mesh"
    
    mkdir -p ${PhysicalHippUnfoldFolder} ${AtlasHippUnfoldFolder}

    log_Msg "Processing $LoopModality $Mesh"
    PostHippUnfold $PhysicalHippUnfoldFolderOut $PhysicalHippUnfoldFolder $AtlasHippUnfoldFolder $T1wFolder $AtlasFolder $Subject $LoopModality $Mesh
```
Here is another possibility to avoid the variable collision problem.